frontend/src/lib/components/icon/Icon.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import type {IconProps} from './Icon.type';
+  import {ICONS} from './icons';
+
+  type $$Props = IconProps;
+  export let name: $$Props['name'];
```

Comment:
Can you try recreating the types? For me it works, but I had that problem at some point.
![Uploading Screenshot 2023-11-06 at 17.38.08.png…]()

If you still don't get them, it might be related to the prop being called `name`. I could change it to `type` for example.

Reply:
it works now